jerry-debugger/jerry-client-ws.py

```diff
@@ -55,6 +55,8 @@
 JERRY_DEBUGGER_WAIT_FOR_SOURCE = 24
 JERRY_DEBUGGER_OUTPUT_RESULT = 25
 JERRY_DEBUGGER_OUTPUT_RESULT_END = 26
+JERRY_DEBUGGER_STHROW = 27
+JERRY_DEBUGGER_STHROW_END = 28
```

Comment:
This function sends a byte, not a string. Actually only two values are allowed. What about sending a stop instead?

Reply:
I've changed the function to send a 0 or 1 only to the client.